src/main/java/SquareHealth/Map/Medicine_User/Controller/DivisionController.java

```diff
@@ -0,0 +1,26 @@
+package SquareHealth.Map.Medicine_User.Controller;
+
+import SquareHealth.Map.Medicine_User.DTO.DivisionDTO;
+import SquareHealth.Map.Medicine_User.Service.DivisionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/div")
+public class DivisionController {
+
+    private final DivisionService divisionService;
+
+    @GetMapping("/division-list")
```

Comment:
@RequestMapping("/div") then your final url  : "/div/division-list
make this url meaning full like "/division/list"